### DISCUSSION TITLE
为什么不能支持vite打包，从原理角度是怎么样的

### DISCUSSION THREAD

**Discussion by dongoa** [alibaba/formily#3902] [Q&A 1.x]
最近写了项目引入formily，试了半天组件交互有问题，伴随报错，搜集网络后看到有人说是vite和formily有兼容性问题，并且在按需加载文档中没有发现vite这一项，想了解不支持的这块底层的原因？以及如何在vite项目中用lerna去管理不同packages达到在vite中使用formily的目的？

**Comment by xxih** [comment]
https://segmentfault.com/q/1010000043540533
<img width="782" alt="image" src="https://github.com/alibaba/formily/assets/64196715/d70f597f-7318-4dd4-b67e-e851829efd95">
试试
